react/tic-tac-toe/src/app/screens/Login/Validations/index.js

```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line no-useless-escape
+const VALID_EMAIL = /^\w+([\.-]?\w+)*@\wolox.com/;
+
+exports.isRequired = value => value ? value : 'Required field';
```

Comment:
You can do this instead

```suggestion
export const isRequired = value => value ? value : 'Required field';
```